app/shared/ui-wizard/src/commonMain/kotlin/ui/wizard/step/SelectTheme.kt

```diff
@@ -0,0 +1,194 @@
+import me.him188.ani.app.ui.settings.tabs.theme.ThemePreviewPanel
+import me.him188.ani.app.ui.theme.themeColorOptions
+import me.him188.ani.app.ui.wizard.WizardLayoutParams
+import me.him188.ani.utils.platform.isAndroid
+
+@Composable
+private fun renderThemeModeText(mode: DarkMode): String {
+    return when (mode) {
+        DarkMode.LIGHT -> "亮色"
+        DarkMode.DARK -> "暗色"
+        DarkMode.AUTO -> "自动"
+    }
+}
+
+@Composable
+internal fun SelectTheme(
+    config: ThemeSettings,
+    onUpdate: (ThemeSettings) -> Unit,
+    modifier: Modifier = Modifier,
+    layoutParams: WizardLayoutParams = WizardLayoutParams.Default
+) {
+    val platform = LocalPlatform.current
+
+    val panelModifier = remember { Modifier.size(96.dp, 146.dp) }
+    val themePanelItem: @Composable (DarkMode) -> Unit = {
+        ColorSchemePreviewItem(
+            onClick = { onUpdate(config.copy(darkMode = it)) },
+            panel = {
+                if (it != DarkMode.AUTO) ThemePreviewPanel(
+                    colorScheme = appColorScheme(isDark = it == DarkMode.DARK),
+                    modifier = panelModifier,
+                ) else DiagonalMixedThemePreviewPanel(
+                    leftTopColorScheme = appColorScheme(isDark = false),
+                    rightBottomColorScheme = appColorScheme(isDark = true),
+                    modifier = panelModifier,
+                )
+            },
+            text = { Text(renderThemeModeText(it)) },
+            selected = config.darkMode == it,
+        )
+    }
+    
+    SettingsTab(modifier = modifier) {
+        Row(
+            modifier = Modifier
+                .padding(horizontal = layoutParams.horizontalPadding)
+                .padding(top = 16.dp)
+                .fillMaxWidth()
+                .horizontalScroll(rememberScrollState())
+                .selectableGroup(),
+            horizontalArrangement = Arrangement.spacedBy(12.dp, Alignment.CenterHorizontally),
+        ) {
+            themePanelItem(DarkMode.LIGHT)
+            themePanelItem(DarkMode.DARK)
+            themePanelItem(DarkMode.AUTO)
+        }
+        Group(
+            title = { Text("色彩") },
+            useThinHeader = true,
+        ) {
+            if (platform.isAndroid()) TextItem(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .clickable {
+                        onUpdate(config.copy(useDynamicTheme = !config.useDynamicTheme))
+                    },
+                title = { Text("动态色彩") },
+                description = { Text("使用桌面壁纸动态生成主题颜色") },
+                action = {
+                    Switch(
+                        checked = config.useDynamicTheme,
+                        onCheckedChange = {
+                            onUpdate(config.copy(useDynamicTheme = !config.useDynamicTheme))
+                        },
+                    )
+                },
+            )
```

Comment:
```suggestion
            if (platform.isAndroid()) { 
                    TextItem(
                        modifier = Modifier
                            .fillMaxWidth()
                            .clickable {
                                onUpdate(config.copy(useDynamicTheme = !config.useDynamicTheme))
                            },
                        title = { Text("动态色彩") },
                        description = { Text("使用桌面壁纸动态生成主题颜色") },
                        action = {
                            Switch(
                                checked = config.useDynamicTheme,
                                onCheckedChange = {
                                    onUpdate(config.copy(useDynamicTheme = !config.useDynamicTheme))
                                },
                            )
                        },
                    )
            }
```

suspecious if usage

注意我 suggest 的可能 indent 不对